app/steps/ui/executors/number/index.js

```diff
@@ -24,13 +26,31 @@ module.exports = class ExecutorsNumber extends ValidationStep {
 
         if (ctx.list.length > ctx.executorsNumber) {
             return {
+                executorsRemoved: slice(ctx.list, 1, ctx.list.length),
                 list: dropRight(ctx.list, ctx.list.length -1),
                 executorsNumber: ctx.executorsNumber
             };
         }
         return ctx;
     }
 
+    * handlePost(ctx) {
+        if (ctx.executorsRemoved && ctx.invitesSent === 'true') {
+            yield ctx.executorsRemoved
+                .filter(exec => exec.isApplying && !exec.isApplicant && exec.inviteId)
+                .map(exec => {
+                    return services.removeExecutor(exec.inviteId)
+                        .then(result => {
+                            if (result.name === 'Error') {
+                                throw new Error('Error while deleting executor from invitedata table.');
+                            }
+                        });
+                });
+        }
+        delete ctx.executorsRemoved;
+        return [ctx];
+    }
```

Comment:
We're also doing this in `app/steps/ui/executors/applying/index.js`, can we create a common method that's used in both places?